inventory/all_projects/pulgit

```diff
@@ -0,0 +1,4 @@
+[pulgit_production]
+git.lib.princeton.edu
+[pulgit_staging]
+git-staging.lib.princeton.edu
```

Comment:
What is the purpose of the staging system? Is it just to test updates to the GitLab software?

Reply:
I have done the migration from svn -> git before. I will want more than one person to verify this process. Testing this on staging will make me feel better.